recipes/augustus/3.2.2/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -x -e
+
+export CC=${PREFIX}/bin/gcc
+export CXX=${PREFIX}/bin/g++
+
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export BOOST_INCLUDE_DIR=${PREFIX}/include
+export BOOST_LIBRARY_DIR=${PREFIX}/lib
+
+export CXXFLAGS="-DUSE_BOOST -I${BOOST_INCLUDE_DIR} -L${BOOST_LIBRARY_DIR}"
+export LDFLAGS="-L${BOOST_LIBRARY_DIR}"
+
+
+make
+mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/scripts
+mkdir -p $PREFIX/config
+mv bin/* $PREFIX/bin/
+mv scripts/* $PREFIX/scripts/
+mv config/* $PREFIX/config/
+
+#Install perl reqs
+cpanm Cwd File::Basename FileHandle File::Path File::Spec::Functions Getopt::Long IO::File List::Util Term::ANSIColor
```

Comment:
What are you thinking? An augustus-deps package?

I'm not sure what the gain is on something like that. Ultimately I want to be to hand my users one command, 

`conda install -c bioconda my-thing`

 and then I want for them to actually be able to use it. If its missing all kinds of dependencies I will have a million tickets for things that aren't installed. 

`conda install -c bioconda my-thing my-thing-deps`

Would also accomplish this. ;-)